app/models/work.rb

```diff
@@ -147,6 +147,15 @@ def check_for_invalid_chapters
   # Run Taggable#check_for_invalid_tags as a validation.
   validate :check_for_invalid_tags
 
+  COMMENT_PERMISSIONS_ENABLE_ALL = 0
+  COMMENT_PERMISSIONS_DISABLE_ANON = 1
+  COMMENT_PERMISSIONS_DISABLE_ALL = 2
+
+  validates_inclusion_of :comment_permissions,
+    in: [COMMENT_PERMISSIONS_ENABLE_ALL,
```

Comment:
Do we want to switch over to this style for arguments? It would be consistent with what we use for method call chains:

```yml
Layout/MultilineMethodCallIndentation:
  EnforcedStyle: indented
```

Reply:
When I tried to switch over to an enum, I deleted this check, so it'll hopefully be a moot point for this PR. (As long as the enum plan doesn't go wrong. 🤞)

For the future ... IDK. I feel like my preference is to align the arguments if they're surrounded by parens, and indent the arguments if they're not surrounded by parens. Which is a very unhelpful preference, because it means that I'm basically indifferent about which style I'd prefer to be applied across the board. But if you think it'd be more consistent to have everything be indented, I'm fine with that.